README.md

```diff
@@ -31,6 +31,13 @@ export INSTALL_MOD_PATH=$TARGET_DIR
 It should install the mali.ko Linux kernel module into the target filesystem,
 and the module should be loaded automatically. If it isn't, modprobe will help.
 
+To compile using parallel build use -p option as follows:
+
+```
+./build.sh -r r6p2 -p 8 -b
+```
+where "-p 8" stays for "-j8"
```

Comment:
I guess we could use the same options name than for make ? aka jobs and -j, we would be consistent that way.

Reply:
Sorry but I've messed up the previous branch and had to recreate it.
This is why there is a new pull request.